src/EnergyPlus/ElectricPowerServiceManager.cc

```diff
@@ -1924,7 +1934,7 @@ Real64 ElectPowerLoadCenter::calcLoadCenterThermalLoad()
         bool plantNotFound = false;
         for (auto &g : elecGenCntrlObj) {
             plantNotFound = false;
-            PlantUtilities::ScanPlantLoopsForObject(g->name,
+            PlantUtilities::ScanPlantLoopsForObject(g->compPlantName,
```

Comment:
Subtle.  OK, so this member variable is going to be set somewhere pretty early on in the code then.